examples/miscellaneous/opacitywindow.py

```diff
@@ -0,0 +1,32 @@
+from kivy.app import App
+from kivy.uix.boxlayout import BoxLayout
+from kivy.lang import Builder
+
+Builder.load_string('''
+#:import window kivy.core.window.Window
+<Layout>:
+    orientation: 'vertical'
+    Label:
+        text: f'Window opacity: {window.opacity}'
+        font_size: 25
+    Slider:
+        size_hint_y: 4
+        min: 0.0
+        max: 1.0
+        value: window.opacity
+        on_value: window.opacity = args[1]
+''')
+
+
+class Layout(BoxLayout):
```

Comment:
`Layout` is a class in `kivy.uix.layout` and we should not re-define it and moreover we don't have to use it. Example can be written like this:
```python
from kivy.app import App
from kivy.lang import Builder

kv = '''
#:import window kivy.core.window.Window

BoxLayout:
    orientation: 'vertical'
    Label:
        text: f'Window opacity: {window.opacity}'
        font_size: 25
    Slider:
        size_hint_y: 4
        min: 0.0
        max: 1.0
        value: window.opacity
        on_value: window.opacity = args[1]
'''


class WindowOpacityApp(App):

    def build(self):
        return Builder.load_string(kv)


if __name__ == '__main__':
    WindowOpacityApp().run()
```

Reply:
Done.